app/components/Terminal.tsx

```diff
@@ -101,7 +101,7 @@ export const Terminal = ({ ws }: TerminalProps) => {
 
   return (
     <>
-      <div className="h-full w-[calc(100%-3rem)]" ref={terminalRef} />
+      <div className="h-full w-[calc(100%-3rem)] text-mono-code" ref={terminalRef} />
```

Comment:
Technically the terminal is styled inline with some values passed to `xterm.js`. By including `text-mono-code` here it's also adding a bunch of styles that are ignored/overriden by those inline styles, but the `font-feature-settings` come with it, so it's probably the easiest and best fix here.